packages/office-ui-fabric-react/src/components/Dialog/DialogContent.tsx

```diff
@@ -35,10 +35,11 @@ export class DialogContent extends BaseComponent<IDialogContentProps, IDialogCon
   public render() {
     let {
       showCloseButton,
-      closeButtonAriaLabel,
+      closeButtonAriaLabel = 'Close',
```

Comment:
If it's standard practice in Fabric now, that's fine; I haven't always kept up with the code base. I just don't want to do it some places and not others, leading to inconsistencies.